src/quo2/components/buttons/predictive_keyboard/view.cljs

```diff
@@ -0,0 +1,67 @@
+(ns quo2.components.buttons.predictive-keyboard.view
+  (:require [react-native.core :as rn]
+            [quo2.components.buttons.predictive-keyboard.style :as style]
+            [quo2.components.info.info-message :as info-message]
+            [react-native.linear-gradient :as linear-gradient]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.buttons.button :as button]))
+
+(def gradients
+  {:light [(colors/alpha colors/neutral-40 0.05) (colors/alpha colors/neutral-40 0)]
+   :dark  [(colors/alpha colors/neutral-80 0.7) (colors/alpha colors/neutral-80 0)]
+   :blur  [colors/white-opa-5 colors/white-opa-0]})
+
+(defn- render-word
+  [word {:keys [blur? on-press]}]
+  [button/button
+   (merge {:type :blur-bg :size 32 :on-press #(on-press word)} (when blur? {:override-theme :dark}))
+   word])
+
+(defn- separator
+  []
+  [rn/view {:width 8}])
+
+(defn view
+  "Options
+   - `type` `:words`/`:error`/`:info`/`:empty`.
+   - `blur?` Boolean to enable blur background support.
+   - `text`  error/info text.
+   - `words` List of words to display in the keyboard.
+   - `on-press` Callback called when a word is pressed `(fn [word])`."
+  [{:keys [type blur? text words on-press]}]
+  [linear-gradient/linear-gradient
+   {:style               (style/wrapper type)
+    :accessibility-label :predictive-keyboard
+    :colors              (if blur?
+                           (gradients :blur)
+                           (colors/theme-colors (gradients :light) (gradients :dark)))}
+
+   (condp = type
```

Comment:
Worth sharing here: in Clojure it's a bit more idiomatic to use `case` when the clauses are compile-time values as is the case here, it's also slightly more performant than `condp` (not that it matters imo, not for our use cases). Just a reminder to also add the default clause, otherwise Clojure will throw an error if no match is found.